index.bs

```diff
@@ -577,66 +577,155 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		<div class="algorithm" data-algorithm="navigator-clipboard">
 		<h4 id="h-navigator-clipboard"><dfn>clipboard</dfn></h4>
 		The [=clipboard=] attribute must return the {{Navigator}}'s [=clipboard=] object.
+		It is used to execute read/write operations from/to the [=system clipboard=].
 		</div>
 
 		</div><!-- dfn-for Navigator -->
 
-	<h3 id="clipboard-interface">Clipboard Interface</h3>
-
+	<h3 id="clipboard-item-interface">ClipboardItem Interface</h3>
 		<pre class="idl" data-highlight="webidl">
-		typedef sequence&lt;ClipboardItem> ClipboardItems;
+			typedef (DOMString or Blob) ClipboardItemDataType;
```

Comment:
In the output, the indentation for this IDL block is all messy; see https://pr-preview.s3.amazonaws.com/w3c/clipboard-apis/pull/158.html#clipboard-item-interface.

I'd expect:

- Indenting stuff inside braces
- Lining up parameter declarations when a definition is split over multiple lines.
- Newline after `[SecureContext, Exposed=Window]`

Reply:
Ack.